fiftyone/server/routes/camera.py

```
@@ -7,19 +7,174 @@
+import fiftyone.core.camera as focam
+import fiftyone.core.groups as fog
 from fiftyone.server import utils
 from fiftyone.server.utils.datasets import get_dataset, get_sample_from_dataset
 
+_MAX_COLLECT_DEPTH = 4
+_SerializedStaticTransform = Dict[str, object]
+
 logger = logging.getLogger(__name__)
 
 
+def _parse_transform_key(key: str):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_parse_transform_key`

(ANN202)